src/backend/setup.cfg

```diff
@@ -26,27 +26,30 @@ requires-python = >=3.6
 
 [options]
 install_requires =
-    boto3==1.9.96
+    boto3==1.9.96 # pyup: <=1.9.96
     coreapi==2.3.3
-    cryptography==2.3.1
-    django==2.0
+    cryptography==2.6.1
+    django==2.1.8 # pyup: >=2.1,<2.2
     dj-database-url==0.5.0
-    django-configurations==2.0
-    django-extensions==2.1.0
+    django-configurations==2.1
+    django-extensions==2.1.6
     # django 2 support from version 1.21, but still alpha release
-    django-postgres-extra==1.21a12
-    djangorestframework==3.8
-    djangorestframework_simplejwt==3.2.3
+    django-postgres-extra==1.21a16
+    djangorestframework==3.9.3
+    djangorestframework_simplejwt==4.3.0
     django-safedelete==0.5.1
     # superior versions of django-storages are not compatible with ManifestStaticFilesStorage
-    django-storages==1.6.3
+    django-storages==1.6.3 # pyup: <=1.6.3
     dockerflow==2018.4.0
-    gunicorn==19.8.1
+    gunicorn==19.9.0
     # Warning from psycopg2 "The psycopg2 wheel package will be renamed from release 2.8" => psyopg2-binary
-    psycopg2-binary==2.7.5
+    psycopg2-binary==2.7.5 # pyup: <=2.7.5
     PyLTI==0.7.0
-    sentry-sdk==0.4.0
+    sentry-sdk==0.7.14
     requests==2.21.0
+    urllib3==1.24.2 # pyup: >=1.21.1,<1.25
+    chardet==3.0.4 # puyup: >=3.0.2,<3.1.0
+    coreapi==2.3.3
```

Comment:
Those are secondary dependencies pinned to prevent requests runtime warnings, right?

Reply:
`coreapi` is need for new django rest framework version. It's an optional dependency.

`urllib3` and `chardet` are for request but without them tests are failing.

Reply:
`coreapi` is already in the requirements. I didn't notice it, sorry.